ros/src/migrave_game_library/game_base.py

```diff
@@ -169,13 +167,18 @@ def evaluate_answer(self, feedback_emotions: Dict[str, str],
 
             self.say_text("Dafür bekommst du einen Stern! Schau mal auf das Tablet.")
             self.audio_play("rfh-koeln/MIGRAVE/Reward2")
-            image = f"{self.correct_answer_count}Token"
+            if self.task == "order_steps":
```

Comment:
I would replace the equality tests with a test that checks if `order_steps` exists in the task name, e.g.
```
self.task.find('order_steps') != -1
```

so that this can work for different ordering games (if there are more of those).